src/plugins/status_page/public/status_page.js

```diff
@@ -28,9 +28,11 @@ const chrome = require('ui/chrome')
 
       const data = resp.data;
       ui.metrics = data.metrics;
-      ui.statuses = data.status.statuses;
       ui.name = data.name;
 
+      ui.statuses = _.groupBy(data.status.statuses, s => s.plugin ? 'plugins' : 'generic');
```

Comment:
The check for `s.plugin` here seems to break abstraction. What do you think of the `Status` class having an `isPluginStatus` method that abstracts away this check, to with similar methods like `createForPlugin` and `getForPluginId` and then using that method here instead?